core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -64,12 +64,11 @@ case class RawlsBillingProject(projectName: RawlsBillingProjectName,
 case class WorkspaceBillingAccount(workspaceName: WorkspaceName, currentBillingAccountOnGoogleProject: Option[RawlsBillingAccountName])
 
 case class RawlsBillingProjectResponse(projectName: RawlsBillingProjectName,
+                                       status: CreationStatuses.CreationStatus,
```

Comment:
IIRC there was some discussion about making this an option to make it easier to remove later(?)
What are your thoughts on that?

Reply:
Not sure - I'm also not sure how making it an `Option` would help with this too. What did you have in mind?

Reply:
Just wanted to raise the question. After v1 billing projects are all migrated to v2 billing projects, this field will no longer be useful. I think if this is an Option it could make it easier to deprecate (by setting this to None) later without needing a new api version but maybe it's not worth the effort 🤷

Reply:
Thanks for doing so! I documented it as deprecated in the swagger page at least. Admittedly I'd have liked to include the version and this as an optional but we can't decipher the billing project version from the database alone and so that ship sailed :'(